client/bugLogService.cfc

```diff
@@ -558,6 +558,10 @@
 	<cffunction name="sanitizeKeyData" access="private" returntype="struct" hint="removes values from structures for known fields that hold sensitive data">
 		<cfargument name="data" type="struct">
 		<cfset var local = {}>
+		<!--- Fix for JSON keys with a null value in ACF 10+ 8 & 9 would set null values to "null" string --->
+            	<cfif !StructKeyExists(arguments.data,local.i)>

```

Comment:
I think this needs to be inside the `<cfloop>` below, since at this point `local.i` is undefined.